src/Caller.cpp

```diff
@@ -36,10 +36,11 @@ using namespace std;
 Caller::Caller() :
     pNorm_(NULL), pCheck_(NULL), protEstimator_(NULL), enzyme_(NULL),
     tabInput_(true), readStdIn_(false), inputFN_(""), xmlSchemaValidation_(true),
-    tabOutputFN_(""), xmlOutputFN_(""), weightOutputFN_(""),
+    tabOutputFN_(""), xmlOutputFN_(""), xmlPeptideOutputFN_(""),weightOutputFN_(""),
```

Comment:
Could you rename this variable to pepXMLOutputFN_. There are other XML formats involved in the nearby code.